src/renderer/features/servers/components/add-server-form.tsx

```diff
@@ -33,11 +33,11 @@ export const AddServerForm = ({ onCancel }: AddServerFormProps) => {
     const form = useForm({
         initialValues: {
             legacyAuth: false,
-            name: '',
+            name: !isElectron() ? window.SERVER_NAME : process?.env.SERVER_NAME?? '' ,
```

Comment:
`process` is not available because `contextIsolation` is true. Consider instead exposing the environment variables in `preload.ts`, which can then be available to the renderer. As-is, this breaks the desktop app